src/plugins/spawnables/spawnables.game.ts

```diff
@@ -191,11 +193,9 @@ export const SpawnablesModule = defineModule<GameState, SpawnablesModuleState>({
 
         module.hitAudioEmitters.set(eid, audioEmitter);
 
-        // const rigidBodyDesc = remote
-        //   ? RAPIER.RigidBodyDesc.newKinematicPositionBased()
-        //   : RAPIER.RigidBodyDesc.newDynamic();
-        const rigidBodyDesc = RAPIER.RigidBodyDesc.newDynamic();
-
+        const rigidBodyDesc = remote
```

Comment:
We repeat this for every rigidbody prefab, we might want to dry this up by making a `remote` parameter that you pass to a `createRigidBody` function?